README.md

```diff
@@ -1,73 +1,19 @@
 ## BDD Testing for "_Schulcloud-Verbund-Software_" using Cypress & Cucumber
 
-Frameworks:
+This project utilizes Cypress and Cucumber for Behavior-Driven Development (BDD) testing. For detailed information on setting up, running tests, and understanding the project structure, please refer to the following documentation:
 
-- [Cypress v13.2.0](https://docs.cypress.io/guides/references/changelog#12-8-0)
+### Documentation
 
-- [Cypress cucumber pre-processor v18.0.6](https://github.com/badeball/cypress-cucumber-preprocessor)
+- **[Setup Guide](./docs/setup.md)**: Instructions for configuring your development environment, including prerequisites, setup, and installation.
+- **[Running Tests Guide](./docs/running_tests_guide.md)**: How to execute tests, including script naming conventions and details for running tests in different environments.
+- **[Tags Documentation](./docs/tags.md)**: Explanation of the tagging system used in our tests, including tag usage, hierarchy, and examples.
+- **[Folder Structure](./docs/folder_structure.md)**: Overview of the project directory layout and the purpose of each folder and file.
+- **[Branch Activation Process](./docs/branch_activation.md)**: Details on the branch activation process and how to extend activation time using labels.
 
-### Prepare development environment
+### Branch Activation Process
 
-Install or make sure Chrome and Edge browsers (at least) have it in your development machine
+To utilize e2e tests in other repositories, you must complete the branch activation process. This includes applying specific labels to PRs to extend the activation time for namespaces.
```

Comment:
May be this line can be with more info--> _This includes applying specific labels to the pull requests (PRs) to extend the activation time for the branch URL of the namespaces (e.g. BRB/NBC/dBC)_ .